testing/adapter/index.js

```diff
@@ -198,10 +199,54 @@ async function nearToEthUnlock ({
   exit(0)
 }
 
+// View
+async function getBridgeOnNearBalance ({
+  nearReceiverAccount,
+  nearErc20Account,
+  nearNetworkId,
+  nearNodeUrl,
+}) {
+  try {
+    // @ts-ignore
```

Comment:
remove
```suggestion
```

Reply:
Ok, got it. I agree in this case it's better to stick to `cli/index.js` approach.